widgetry/src/widgets/spinner.rs

```diff
@@ -1,8 +1,9 @@
+use geom::{Polygon, Pt2D};
```

Comment:
I'm curious, did cargo fmt or your editor make this change? I haven't found options for cargo fmt to automate this kind of thing. I'd prefer being consistent with the rest of the code, so just keeping all the imports in one paragraph, not separating external crates and the current crate.

Reply:
That'd be my editor, clion with the rust extension. Just using alt enter on a piece of code to import the relevant library. 
But I'll change it back

Reply:
What would you prefer the import to be?
Because it's the same in [button widget](https://github.com/dabreegster/abstreet/blob/1079a899d60c124bc40dbe69ca66cc58f48e81f9/widgetry/src/widgets/button.rs#L5) ?

Reply:
Ah, automatic import is a nice feature. I don't use an IDE, so I'm stuck waiting for https://github.com/rust-lang/rustfmt/issues/2979. I could hack something together to canonicalize imports if manually reverting gets annoying.

Reply:
Yeah, that's annoying and seems it won't be fixed for a while :(